### PR TITLE
new category entity

### PR DESCRIPTION
**Pull request by dobby-ide**
let me know if this is what we need

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,19 @@
...
+#### Category entity
+enum Categories = {
+	fantasy = 'fantasy'
+	biography = 'bio'
+	sport = 'sport'
+	real-life = 'real-life'
+	computer-science = 'computer science'
+}
+{
+	"id":number,
```
Comment (Surtt) [inline README.md:17]: I think at this stage we need just something like this:
`{
	"id":number,
	"categoryName":string
}`

Comment (dobby-ide) [reply README.md:11]: done!

File: README.md [hunk 2]
```diff
@@ -5,11 +5,22 @@
...
 
 #### Books endpoints
 ```Books endpoints here```
 
 #### Users endpoints
 ```Users endpoints here```
 
+POST /api/v1/user/login/
+POST /api/v1/user/logout/
+PUT /api/v1/user/update/{userId}
```
Comment (Surtt) [inline README.md:24]: As Yazan mentioned we shouldn't use words like update/create etc.
Снимок экрана 2023-04-04 в 17 06 18

I think it's better this way
```suggestion
POST /api/v1/users/login
POST /api/v1/users/logout
PUT /api/v1/users/{userId}
```

File: README.md [hunk 3]
```diff
@@ -5,11 +5,25 @@
...
+
 
 #### Books endpoints
 ```Books endpoints here```
 
 #### Users endpoints
 ```Users endpoints here```
 
+POST /api/v1/user/login/
+POST /api/v1/user/logout/{token}
```
Comment (Surtt) [inline README.md:23]: It would be better to use plural form like: 
POST /api/v1/**users**/logout/{token}

what is token used for here?

Comment (dobby-ide) [reply README.md:23]: token of the login session. for the server to delete it

File: userEndpoint.yaml [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  - name: store
+    description: Access to Petstore orders
+    externalDocs:
+      description: Find out more about our store
+      url: http://swagger.io
+  - name: user
+    description: Operations about user
+paths:
+  /api/v1/user/login:
+    put:
```
Comment (Surtt) [inline userEndpoint.yaml:32]: Here should be POST

File: README.md (hunk 3 above)
Comment (Surtt) [reply README.md:23]: Endpoints have not been changed,
It should be **plural** :)

**Pull request closed without merging**